Loop/Managers/LoopDataManager.swift

```diff
@@ -1155,25 +1155,6 @@ extension LoopDataManager {
         )
     }
 
-    fileprivate func computeCarbsOnBoard(potentialCarbEntry: NewCarbEntry?, replacing replacedCarbEntry: StoredCarbEntry?) -> CarbValue? {
```

Comment:
All of the `computeCarbsOnBoard` code deleted here is cruft from an older bolus screen implementation.